src/store/modules/accounts/index.js

```diff
@@ -1,10 +1,9 @@
 /* eslint no-param-reassign: ['error', { 'ignorePropertyModificationsFor': ['state'] }] */
 
 import hdWallet from './hdWallet';
-import ledger from './ledger';
 import { ACCOUNT_HD_WALLET } from '../../../popup/utils';
 
-const modules = { hdWallet, ledger };
+const modules = { hdWallet };
```

Comment:
I suggest to add a TODO comment to describe why this is an object and that we need to remove the concept of "modules" here.